site/class-1-play/mausnl.svg

```diff
@@ -0,0 +1,15 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0, 0, 500, 500">

```

Comment:
Your SVG is still broken:

```
  1:1  warning  3.0. `<rect>` element                          4   lint
  1:1  warning  3.1. `x` attribute on `<rect>`                 5   lint
  1:1  warning  3.2. `y` attribute on `<rect>`                 6   lint
  1:1  warning  3.3. `width` attribute on `<rect>`             7   lint
  1:1  warning  3.4. `height` attribute on `<rect>`            8   lint
  1:1  warning  7.3. Content in `<text>` element               23  lint
  1:1  warning  8.3. `<g>` element with at least two children  27  lint
  1:1  warning  10.1. `font-family` attribute on any element   30  lint
  1:1  warning  10.2. `font-size` attribute on any element     31  lint

Reply:
You can also remove the file here if you’re adding it in another PR.

You can remove files through the github interface, by going to your fork, your branch, and the finding your file. Finally, click on the trash icon in the top-right.